docs/docs/releases/0.2.0.md

```diff
@@ -244,4 +244,114 @@ We also set up a workflow to reset the live demo regularly to mitigate reliabili
 
 ## Upgrading to 0.2.0 {:#upgrading}
 
-We do not support upgrading from previous versions to `0.2.0`. You will need to install a fresh instance of Mathesar to use this version.
+We do not fully support upgrading from previous versions to `0.2.0`. In particular, you will lose any Explorations and User accounts (e.g., usernames and passwords) you created in Mathesar during the process. Furthermore, manual steps are required.
+
+### Prerequisite steps (run before starting Mathesar version 0.2.0)
+
+If you want to reuse the same database server for Mathesar's internal database, you need to drop the `mathesar_django` database and recreate it on that server, using the same PostgreSQL role that you initially used to install Mathesar, or a role with superuser privileges.
+
+#### If there is another database on the server
+
+If there is another PostgreSQL database on the same server, first log into that other database, and then run:
+
+```sql
+DROP DATABASE mathesar_django;
+CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
+```
+
+#### If there is not another database on the server
+
+If there is **not** another database on the server, it's more difficult, since you can't drop the only database on a PostgreSQL server. First, log into the `mathesar_django` database, and run:
+```sql
+CREATE DATABASE mathesar_temp;
+```
+Log out, and log into the `mathesar_temp` database. Then run:
+```sql
+DROP DATABASE mathesar_django;
+CREATE DATABASE mathesar_django OWNER mathesar;  -- or whichever role you use to run Mathesar
+```
+Log out, and log into the `mathesar_django` database. Then run:
+```sql
+DROP DATABASE mathesar_temp;
+```
+
+After this, you're ready to proceed with the upgrade.
+
+### For installations using Docker Compose
+
+If you have a Docker compose installation (including one from the guided script), run the command below:
+
+```
+docker compose -f /etc/mathesar/docker-compose.yml up --pull always -d
```

Comment:
@mathemancer  We would be moving to a different repository in Docker i.e. `mathesar/mathesar` instead of `mathesar/mathesar-prod`, so this wouldn't work.

We would have to ask users to modify their docker-compose file or get the latest one from our repo.

Reply:
Good catch, thanks